WalletWasabi/WabiSabi/Client/CoinJoin/Manager/CoinJoinManager.cs

```diff
@@ -211,8 +211,8 @@ async Task<IEnumerable<SmartCoin>> SanityChecksAndGetCoinCandidatesFunc()
 						throw new CoinJoinClientException(CoinjoinError.AllCoinsPrivate);
 					}
 
-					// If coin candidates are already private and the user doesn't override the StopWhenAllMixed, then don't mix.
-					if (coinCandidates.All(x => !x.Confirmed || x.IsPrivate(walletToStart.AnonScoreTarget)) && startCommand.StopWhenAllMixed)
```

Comment:
`!x.Confirmed` can be ignored entirely. `SelectCandidateCoinsAsync` removes all the unconfirmed coins from the list. So it is not possible to have unconfirmed coins here. It will be false every time, so it does not matter in OR condition.

Reply:
Where does `SelectCandidateCoinsAsync` remove all the unconfirmed coins from the list? I looked at the path and couldn't see it